apps/src/sites/studio/pages/code-studio.js

```diff
@@ -77,6 +79,18 @@ window.onerror = function (msg, url, ln) {
   }
 };
 
+// Prevent filtered errors from being passed to New Relic.
+if (window.newrelic) {
+  window.newrelic.setErrorHandler(function (err) {
+    // Remove errors from unsupportenewrelicnd IE versions
+    if (isUnsupportedBrowser()) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Could just be `return isUnsupportedBrowser()`? or if that might return things that aren't bools, `return !!isUnsupportedBrowser()`